README.md

```diff
@@ -7,6 +7,35 @@ A simple interface for the HathiTrust APIs. The package contains basic classes a
 [data api]: http://www.hathitrust.org/data_api
 [solr api]: http://wiki.htrc.illinois.edu/display/COM/Solr+Proxy+API+User+Guide
 
+## UPDATES in Fork
```

Comment:
These comments are very useful, but shouldn't be in the README - they belong in the commit message.